app/src/main/java/org/mozilla/fenix/tabstray/browser/TabGroupViewHolder.kt

```diff
@@ -59,8 +59,10 @@ class TabGroupViewHolder(
 
             adapter = groupListAdapter
 
-            groupListAdapter.submitList(group.tabs)
-            scrollToPosition(selectedIndex)
+            groupListAdapter.submitList(tabGroup.tabs)
+            selectedIndex.let {
+                scrollToPosition(it)
+            }
```

Comment:
Forgot to tell you later on that I found the code I was looking for about the default value being `NO_POSITION` (-1): https://cs.android.com/androidx/platform/frameworks/support/+/androidx-main:recyclerview/recyclerview/src/main/java/androidx/recyclerview/widget/RecyclerView.java;l=277

So we don't need to do extra checks because [we're covered](https://cs.android.com/androidx/platform/frameworks/support/+/androidx-main:recyclerview/recyclerview/src/main/java/androidx/recyclerview/widget/LinearLayoutManager.java;l=540).

In fact, the linter tells we should remove this either way.

<img width="426" alt="Screen Shot 2021-09-22 at 10 04 32 PM" src="https://user-images.githubusercontent.com/1370580/134444768-c8de7a77-8aa3-4f60-a161-db1d397e1a9e.png">